server/src/main/java/org/candlepin/controller/OwnerManager.java

```diff
@@ -133,19 +131,16 @@ public void cleanupAndDelete(Owner owner, boolean revokeCerts) {
             permissionCurator.delete(perm);
         }
 
-        for (Product p : this.ownerProductCurator.getProductsByOwner(owner)) {
-            log.info("Deleting product: {}", p);
-            this.prodManager.removeProduct(p, owner);
-        }
+        log.info("Deleting all products for owner: {}", owner);
+        this.productManager.removeAllProducts(owner);
 
-        /*
-         * contents might have been deleted due to cascades above.
-         */
-        ownerContentCurator.flush();
-        for (Content c : ownerContentCurator.getContentByOwner(owner)) {
-            log.info("Deleting content: {}", c);
-            this.contentManager.removeContent(c, owner, false);
-        }
+        log.debug("REMAINING PRODUCTS A: {}", this.ownerProductCurator.getProductsByOwner(owner).list());
```

Comment:
Whoops. These two debug statements were never intended to hit master. I'll remove them.